components/UnitCustomPipeline.vue

```diff
@@ -59,25 +62,27 @@ export default {
     }
   },
   methods: {
-    async runPipeline() {
+    runPipeline() {
       this.message = ''
       this.error = false
       this.progress = true
-      try {
-        const { headers, items } = await this.customPipeline(
-          this.fileManager,
-          this.parameter
-        )
-        this.$emit('update', { headers, items })
-      } catch (error) {
-        console.error(error)
-        this.error = true
-        this.message = processError(error)
-        this.$emit('update', { error })
-      } finally {
-        this.status = true
-        this.progress = false
-      }
+      setTimeout(async () => {
```

Comment:
I found a solution: set the heavy computation to start after 1 ms using setTimeout, this allows the dom to be updated.

Reply:
Nice, thank you !

Reply:
Sounds like a use case for [$nextTick()](https://vuejs.org/v2/api/#Vue-nextTick)

```
await this.$nextTick()
```